frappe/__init__.py

```diff
@@ -1915,13 +1915,13 @@ def as_json(obj: dict | list, indent=1, separators=None) -> str:
 
 	try:
 		return json.dumps(
-			obj, indent=indent, sort_keys=True, default=json_handler, separators=separators
```

Comment:
Don't just enable this globally? Add a flag which defaults to current behaviour and change it where it's required.

Reply:
Ok. By the time being my only use case si while exporting JSON files, so I added the flag in `.as_json()` with default behavior and changed it on `data_import.export_json()`